react/features/conference/components/Conference.native.js

```diff
@@ -290,6 +292,18 @@ class Conference extends Component<Props> {
         return this._backHandler && this.props._onHardwareBackPress();
     }
 
+    /**
+     * Renders the conference notification badge if the feature is enabled.
+     *
+     * @private
+     * @returns {React$Node}
+     */
+    _renderConferenceNotification() {
+        return ConferenceNotification
```

Comment:
is it a legit way of checking if a thing is enabled ? or are you trying to avoid an export and it needs to be this way ?
if that's the case then explanatory comment mentioning that we don't want to export if disabled would be helpful.

Reply:
Yeah, the idea behind disabling the calendar feature is that it doesn't register its middleware and reducer, and also it doesn't export its components (the latter one was Lyubo's idea), so this type of check seems to be necessary in some cases.

But good point about the comment, I add that!